lib/autoproj/manifest.rb

```diff
@@ -60,6 +60,10 @@ def load(file)
                 YAML.safe_load(File.read(file)) || {}
             end
 
+            if data["layout"].member?(nil)
```

Comment:
You don't need the test here. Just call compact.

Reply:
Actually added the test in order to print a warning and then decided to go without it :D

Gonna add a brief User warning to the PR.